buildSrc/src/main/java/com/uber/okbuck/extension/JetifierExtension.java

```diff
@@ -1,13 +1,72 @@
 package com.uber.okbuck.extension;
 
+import com.google.common.collect.ImmutableSet;
+import com.uber.okbuck.core.dependency.ExternalDependency;
+import com.uber.okbuck.core.manager.JetifierManager;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
+import org.gradle.api.Project;
+
 public class JetifierExtension {
 
   public static final String DEFAULT_JETIFIER_VERSION = "1.0.0-beta02";
+  private static final List<String> BLACKLISTED_DEPS =
```

Comment:
Would make sense to provide this as an external piece of configuration? Otherwise, everytime that we figure out that a dep need to be blacklisted okbuck version will have to be bumped right?

Reply:
lets rename to JETIFIER_DEPS and addd javadoc to mirror what your comment said